src/Components/OATPropertyEditor/OATPropertyEditor.styles.ts

```diff
@@ -226,8 +224,8 @@ export const getPropertyEditorTextFieldStyles = () => {
 
 /* Template Column */
 
-export const getTemplateColumnStyles = () => {
-    const theme = useTheme();
+export const getTemplateColumnStyles = (args: { theme: Theme }) => {
```

Comment:
shouldn't the type be ExtendedTheme?

Reply:
this one didn't need our extended theme so I left it as just a basic theme. I could change it, but it doesn't need it.